app/controllers/admin/environments_controller.rb

```diff
@@ -0,0 +1,52 @@
+class Admin::EnvironmentsController < ApplicationController
+  before_action :authorize_admin!
+  before_action :authorize_super_admin!, only: [ :create, :new, :edit, :update, :destroy ]
+  before_action :environment, only: [:edit, :update, :destroy]
```

Comment:
Yup, I was in 2 minds as to whether to leave it there. But decided it would remove a few lines that @grosser can comment on! :smile: